src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,79 @@
-import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
```

Comment:
Maybe type, not interface?
![Screenshot_2](https://user-images.githubusercontent.com/110488158/206713828-e8cb0aa9-974c-424b-9a75-dd7601b486ef.png)

Reply:
some mentors say: "use the interface for PROPS because these are objects", other sources: "use the type"))) Where to look for the truth?)))

Reply:
I always use interface for props and have no problems with that. Why type is more constrained?